ui/src/chat/ChatContent/ChatContent.tsx

```diff
@@ -122,6 +124,14 @@ export function InlineContent({ story }: InlineContentProps) {
     );
   }
 
+  if (isBlockCode(story)) {
+    return (
+      <pre className="bg-gray-50 px-4">
```

Comment:
Seems like this should be `rounded bg-gray-50 py-2 px-4` and no need to do the checking in ChatContent

Reply:
It *should* work like that, but there's a bug in tiptap that creates multiple lines for code blocks:
https://github.com/ueberdosis/tiptap/issues/1585